content/efcore/getting-started/samples/loading-related-data-tutorial/count-method/README.md

```diff
@@ -0,0 +1,10 @@
+# Count Method
+
+This code example demonstrates how to use EF Core to query related data from the database using the .Count method.
+
+## Code Files:
+- [Program.cs](Program.cs)
+- [LibraryContext.cs](../../shared/LibraryContext.cs)
+- [Book.cs](../../shared/Book.cs)
+- [Author.cs](../../shared/Author.cs)
+- [Edition.cs](../../shared/Edition.cs)
```

Comment:
why do all of these have READMEs? Why can't we just include a README in the directory above that says the classes that are included?